lib/go/thrift/simple_server_test.go

```diff
@@ -154,3 +158,112 @@ func TestNoHangDuringStopFromDanglingLockAcquireDuringAcceptLoop(t *testing.T) {
 	runtime.Gosched()
 	serv.Stop()
 }
+
+func TestNoHangDuringStopFromClientNoDataSendDuringAcceptLoop(t *testing.T) {
+	ln, _ := net.Listen("tcp", "localhost:0")
```

Comment:
don't discard errors, even in a test, do something like this instead:
```suggestion
	ln, err := net.Listen("tcp", "localhost:0")
	if err != nil {
		t.Fatalf("Failed to listen: %v", err)
	}
```

same for the other test function.